Jenkinsfile

```diff
@@ -161,7 +161,7 @@ spec:
   }
 
   options {
-    buildDiscarder(logRotator(numToKeepStr: '10'))
+    buildDiscarder(logRotator(numToKeepStr: '2'))
```

Comment:
You can keep 10 builds, but you should only keep the build artifacts of 2 builds:

```
buildDiscarder(logRotator(numToKeepStr: '10', artifactNumToKeepStr: '2'))
```

Reply:
Lets try to limit both, if we would have issues with that, we can always reconfigure it again. The thing is that logs (server and ant logs) are also artifacts here, and without them the build result is not much useful. However as all tests are usually reproducible locally, we don't need to keep so much data on Jenkins.